src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -4440,6 +4467,22 @@ subroutine Summary_carbonflux(this, &
          this%gpp_patch(bounds%begp:bounds%endp), &
          this%gpp_col(bounds%begc:bounds%endc))
 
+    if (use_mimics_decomp) then
+       ! Calculate ligninNratioAve
+       ! TODO define new var cwdc_to_litter as the flux from cwdl2 transition
```

Comment:
I haven't come across anywhere, yet, the variable that represents the flux from the cwdl2 transition. I made a placeholder with cwdc_to_litter(c) for now.

Reply:
I think I found this now (see next commit).